src/main/scala/dpla/ingestion3/mappers/providers/MtMapping.scala

```diff
@@ -87,6 +89,7 @@ class MtMapping extends Mapping[NodeSeq] with XmlExtractor with IdMinter[NodeSeq
   override def subject(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
   // <mods:subject><mods:topic>
     extractStrings(data \\ "subject" \ "topic")
+      .flatMap(_.splitAtDelimiter(";"))
```

Comment:
@moltude what is the difference between `split` and `splitAtDelimiter`?

Reply:
applies trim and filter 
```
    lazy val splitAtDelimiter: (String) => Array[String] = (delimiter) => {
      value.split(delimiter).map(_.trim).filter(_.nonEmpty)
    }```

Reply:
👍